packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -220,6 +220,7 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 				newRevision,
 				this.detachedRevision,
 			);
+			this.editManager.advanceMinimumSequenceNumber(newRevision);
```

Comment:
So not only were we over-summarizing, but we also weren't evicting in the first place for detached. So this PR is two fixes in one, yeah? Awesome!

Reply:
Yup :)